src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/SymbolIsBannedAnalyzer.cs

```diff
@@ -181,7 +179,8 @@ private void OnCompilationStart(CompilationStartAnalysisContext compilationConte
             {
                 var query =
                     from additionalFile in compilationContext.Options.AdditionalFiles
-                    where StringComparer.Ordinal.Equals(Path.GetFileName(additionalFile.Path), SymbolIsBannedAnalyzer.BannedSymbolsFileName)
+                    let fileName = Path.GetFileName(additionalFile.Path)
+                    where fileName != null && fileName.StartsWith("BannedSymbols.", StringComparison.Ordinal) && fileName.EndsWith(".txt", StringComparison.Ordinal)
```

Comment:
Shall we still use consts instead of hardcoded strings?

Reply:
I could introduce `FileNamePrefix`/`FileNameSuffix` consts, but these would be tied to the implementation (since they would include the periods) and would only be referenced once. I don't think that would be a good design.